packages/website/vitePlugins/pluginRoutes/index.ts

```diff
@@ -0,0 +1,59 @@
+import path from "node:path";
+import { fileURLToPath } from "node:url";
+import type { Plugin } from "vite";
+import type React from "react";
+import routesJSON from "../../src/routes.json";
+
+export interface Route {
+  path: string;
+  element: React.ReactElement;
+}
+
+const CONVENTIONAL_ROUTE_ID = "website:routes";
+
+const __dirname = path.dirname(fileURLToPath(import.meta.url));
+
+const menuGroup = routesJSON.main;
+const routeObjects = menuGroup.reduce((pre: string[], cur) => {
+  pre.push(...cur.items);
+  return pre;
+}, []);
+
+const generateRoutesCode = () => {
+  return `
+  import React, {lazy} from 'react'
+
+  ${routeObjects
+    .map((route, index) => {
+      const importPath = path.resolve(__dirname, `../../node_modules/@reactuses/core/hooks/${route}/README.md`);
```

Comment:
`importPath` does not work on windows. The separator is resolved to '\\' and we need to replace it with '/'.

Reply:
Well. I don't have a windows computer. Should I use `path.sep`?

Reply:
```ts
const __dirname = path.posix.dirname(fileURLToPath(import.meta.url));
...
const importPath = path.posix.resolve(__dirname, `../../node_modules/@reactuses/core/hooks/${route}/README.md`);
```